options.go

```diff
@@ -173,37 +178,50 @@ func Statter(s log.StatsReporter) Option {
 	}
 }
 
-// Identity is used to specify a static hostport string as this Ringpop
-// instance's identity.
+// Identity is used to specify an identity as this Ringpop instance's identity.
```

Comment:
Help text could be better here.

E.g.:

```
// Identity can be used to specify a custom string as the unique identifier for
// this node. The identity should be unique amongst other Ringpop instances; it
// is used in the hashring.
//
// By default, the hostport/address of the node is used as the identity in the
// hashring. An error is thrown if a hostport is manually specified using this
// option, as this would lead to unexpected behaviour. If you want to override
// the node's listening address, use the `Address` option.
```